src/components/Avatar/Avatar.module.css

```
@@ -0,0 +1,35 @@
+:root {
+  --small: 48px;
+  --medium: 64px;
+  --large: 96px;
+  --ratio: 0.5;
+}
+
+.container {
+  background: #999999;
```

✋ Since `:root` corresponds to the `html` element, this essentially sets these as global variables. With names as generic as these, I can see there being potential conflicts down the road. What if we scope these to the `.container` element?

```suggestion
.container {
  --small: 48px;
  --medium: 64px;
  --large: 96px;
  --ratio: 0.5;
  background: #999999;
```

---

Yeah, great callout, I wasn't sure whether this would scope or not.